src/Features/Core/Portable/Completion/CompletionHelper.cs

```diff
@@ -43,14 +43,13 @@ public static CompletionHelper GetHelper(Document document)
         /// results, or false if it should not be.
         /// </summary>
         public bool MatchesPattern(string text, string pattern, CultureInfo culture)
-            => GetMatch(text, pattern, culture) != null;
+            => GetMatch(text, pattern, includeMatchSpans: false, culture) != null;
 
-        private PatternMatch? GetMatch(string text, string pattern, CultureInfo culture)
-            => GetMatch(text, pattern, includeMatchSpans: false, culture: culture);
-
-        private PatternMatch? GetMatch(
-            string completionItemText, string pattern,
-            bool includeMatchSpans, CultureInfo culture)
+        public PatternMatch? GetMatch(
```

Comment:
>PatternMatch? [](start = 15, length = 13)

Now or later we may want to remove ?. Something like add PatternKind.NoMatch and use it as default PatternMatch. Maybe not now.